Android/app/src/main/res/values/strings.xml

```diff
@@ -30,6 +30,16 @@
     <string name="signup">Signup</string>
     <string name="toast_invalid_username_or_password">Invalid username or password</string>
 
+    <!-- reset password strings -->
+    <string name="submit">Submit</string>
+    <string name="text_back_to_login"><u>Back to login</u></string>
+    <string name="text_forgot_password"><u>Forgot password? Clear here to reset.</u></string>
```

Comment:
Change `Clear` to `Click`

Reply:
Fixed the typo!
Added javadoc comments for the newly added methods.